core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -279,6 +279,10 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/ExplorationPlayerStateService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/PlayerConstants.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/AnswerClassificationService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/StateClassifierMappingService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/PredictionAlgorithmRegistryService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/PredictionAlgorithmSampleService.js"></script>
```

Comment:
This part is slightly tricky. @seanlip Is there anyway we can do dynamic inclusion here? Like only include the service that is returned by the registry service?